src/Microsoft.IdentityModel.Tokens/BaseConfigurationManager.cs

```diff
@@ -59,6 +55,34 @@ public TimeSpan AutomaticRefreshInterval
         /// </summary>
         public static readonly TimeSpan DefaultRefreshInterval = new TimeSpan(0, 0, 5, 0);
 
+        /// <summary>
+        /// The default constructor.
+        /// </summary>
+        public BaseConfigurationManager()
```

Comment:
I think you can write :this(new LKGConfigurationCacheOptions())

Reply:
Yeah, good suggestion, thanks.